preprocessing/nextclade/src/loculus_preprocessing/new_config.yaml

```diff
@@ -0,0 +1,30 @@
+organism: "enterovirus"
```

Comment:
@corneliusroemer how do you like this config structure for the multi-segmented case?

I've started adding TODO where we need to adapt prepro